### PR TITLE
Bumped version from 2.0.1 to 2.1.0.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
Signed-off-by: AWSHurneyt <hurneyt@amazon.com>

### Description
Bumped version from 2.0.1 to 2.1.0.
 
### Issues Resolved
https://github.com/opensearch-project/alerting-dashboards-plugin/issues/276
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/workflows/cypress-workflow.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ on:
       - 2.*
 env:
   OPENSEARCH_DASHBOARDS_VERSION: '2.0'
```
Comment (lezzago) [inline .github/workflows/cypress-workflow.yml:14]: We need to change this to 2.x

**Review by lezzago — CHANGES_REQUESTED** [review]

File: .github/workflows/cypress-workflow.yml (hunk 1 above)
Comment (AWSHurneyt) [reply .github/workflows/cypress-workflow.yml]: Updated.

**Review by lezzago — APPROVED** [review]

**Review by rishabhmaurya — APPROVED** [review]

**Comment by bowenlan-amzn** [comment]
Seems we miss to change the dashboard version in UT workflow https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/.github/workflows/unit-tests-workflow.yml#L14

**Comment by AWSHurneyt** [comment]
> Seems we miss to change the dashboard version in UT workflow https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/.github/workflows/unit-tests-workflow.yml#L14

Thank you for catching this! I adjusted the branch referenced by this file in [this PR](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/278.)

**Pull request merged**